### PR TITLE
near/overFlow: check for overflow

### PR DESCRIPTION
**Pull request by jumpsiegel**

### CODE REVIEW

**Review by kcsongor — DISMISSED** [review]

File: near/contracts/token-bridge/src/lib.rs [hunk 1]
```diff
@@ -974,6 +974,15 @@ impl TokenBridge {
         if namount == 0 {
             env::panic_str("EmptyTransfer");
         }
+
+        if nfee >= namount {
```
Comment (panoel) [inline near/contracts/token-bridge/src/lib.rs:978]: Everywhere else you make this check last.  Why is it first here?  (Sorry OCD kicking in...)
Does it even matter?

File: near/test/sdk.ts [hunk 2]
```diff
@@ -528,7 +544,7 @@ async function testNearSDK() {
       core_bridge,
       token_bridge,
       randoToken,
-      BigInt(10000000),
+      BigInt(21) * BigInt("1000000000") * BigInt("1000000000000000000"),
```
Comment (panoel) [inline near/test/sdk.ts:547]: Is there something special about 21?

**Review by panoel — DISMISSED** [review]

File: near/test/sdk.ts (hunk 2 above)
Comment (jumpsiegel) [reply near/test/sdk.ts:547]: sweatcoin has 21 billion minted...

File: near/contracts/token-bridge/src/lib.rs (hunk 1 above)
Comment (jumpsiegel) [reply near/contracts/token-bridge/src/lib.rs:978]: no particular reason...

**Review by bruce-riley — DISMISSED** [review]

**Review by bruce-riley — APPROVED** [review]

**Review by kcsongor — APPROVED** [review]

**Pull request merged**